gcloud-java-dns/src/main/java/com/google/gcloud/dns/DnsBatch.java

```diff
@@ -129,13 +131,82 @@ DnsOptions options() {
    */
   public DnsBatchResult<ProjectInfo> getProject(Dns.ProjectOption... options) {
     DnsBatchResult<ProjectInfo> result = new DnsBatchResult<>();
-    DnsRpc.Callback<Project> callback = createProjectCallback(result);
-    Map<DnsRpc.Option, ?> optionMap = optionMap(options);
+    RpcBatch.Callback<Project> callback = createProjectCallback(result);
+    Map<DnsRpc.Option, ?> optionMap = DnsImpl.optionMap(options);
     batch.addGetProject(callback, optionMap);
     return result;
   }
 
-  // todo(mderka) implement remaining operations
+  /**
+   * Adds a request representing the "list record sets" operation in the zone specified by {@code
```

Comment:
Nit. Here and elsewhere, I feel like rephrasing this as:
"Adds a request to this batch representing the "list record sets" operation in the zone specified by zoneName"
Would be slight more clear. Feel free to ignore this comment if you don't agree.